### PR TITLE
DOC-2828 Moved Auto Tiering install to install on Linux.

### PR DESCRIPTION
**Pull request by amberdoctor-redis**
https://redislabs.atlassian.net/browse/DOC-2828

Preview: https://docs.redis.com/staging/DOC-2828/rs/installing-upgrading/install/install-on-linux/

and this page should be deleted: https://docs.redis.com/staging/DOC-2828/rs/installing-upgrading/install/prepare-install/prepare-flash/

### CODE REVIEW

File: content/rs/installing-upgrading/install/install-on-linux.md [hunk 1]
```diff
@@ -5,7 +5,10 @@ description: Install Redis Enterprise Software on Linux.
 weight: 10
 alwaysopen: false
 categories: ["RS"]
-aliases: 
+aliases: [
+ "/rs/installing-upgrading/install/prepare-install/prepare-flash",
+ "/rs/installing-upgrading/install/prepare-install/prepare-flash.md",
```
Comment (rrelledge) [inline content/rs/installing-upgrading/install/install-on-linux.md:10]: We can skip the ".md" alias
```suggestion
```

File: content/rs/installing-upgrading/install/prepare-install/_index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Before you install Redis Enterprise Software:
 
 - If you require the `redislabs` UID (user ID) and GID (group ID) numbers to be the same on all the nodes, create the `redislabs` user and group with the required numbers on each node.
 
-- If you want to use Auto Tiering for your databases, see [Prepare Auto Tiering]({{<relref "/rs/installing-upgrading/install/prepare-install/prepare-flash">}}).
+- If you want to use Auto Tiering for your databases, see [Prepare Auto Tiering]({{<relref "rs/installing-upgrading/install/install-on-linux#prepare-flash">}}).
```
Comment (rrelledge) [inline content/rs/installing-upgrading/install/prepare-install/_index.md:27]: Fixing broken anchor
```suggestion
- If you want to use Auto Tiering for your databases, see [Auto Tiering installation]({{<relref "rs/installing-upgrading/install/install-on-linux#auto-tiering-installation">}}).
```

File: content/rs/installing-upgrading/install/install-on-linux.md [hunk 3]
```diff
@@ -82,6 +85,25 @@ To skip the installation questions, use one of the following methods:
 
 1. Repeat this process for each node in the cluster.
 
+
+## Auto Tiering installation
+
+If you want to use Auto Tiering for your databases, review the prerequisites, storage requirements, and [other considerations]({{< relref "/rs/databases/auto-tiering/" >}}) for Auto Tiering databases and prepare and format the flash memory.
+
+After completing the [Install on Linux]({{< relref "rs/installing-upgrading/install/install-on-linux#install-on-linux" >}}), use the `prepare_flash` script to prepare and format flash memory:
```
Comment (rrelledge) [inline content/rs/installing-upgrading/install/install-on-linux.md:93]: Replacing the old relref with an in-page anchor link
```suggestion
After you [install on Linux](#install-on-linux), use the `prepare_flash` script to prepare and format flash memory:
```

**Comment by kurtfm** [comment]
@rrelledge committed suggestions

**Review by rrelledge — APPROVED** [review]

**Pull request merged**